code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -192,7 +194,15 @@
 
 /obj/machinery/capture_the_flag/attack_ghost(mob/user)
 	if(ctf_enabled == FALSE)
+		if(user.client && user.client.holder)
+			var/response = alert("Enable CTF?", "CTF", "Yes", "No")
+			if(response == "Yes")
+				for(var/obj/machinery/capture_the_flag/CTF in machines)
+					CTF.start_ctf()
+				message_admins("[key_name_admin(user)] has enabled CTF!")
```

Comment:
It's not a proc, it's a section of code in the giant Topic() in /datum/admin. I could procify it if you'd prefer.